content/review.rst

```diff
@@ -306,6 +321,31 @@ Reviewer's check list - QA
   functional tests?
 
 
+Reviewer's check list - Support
+-------------------------------
+
+* Does the documentation contain old information that needs to be updated?
+
+* Does the documentation makes sense to new and existing users?
+
+* UX: Does the Local Manager make sense along with the text configuration?
+
+* If it's a new feature release, does the information make sense to new
+  users?
+
+* Should there be an accompanying Users Guide? FAQ update? API addition?
+
+* Should there be samples, screenshots available?
+
+After a release:
```

Comment:
maybe this needs to be put into the http://styleguide.chevah.com/release-process.html#release-review-process ... after a release, usually mean after a review and after the code is already approved.

this page should talk about what needs to be done to approve a change